public/index.html

```diff
@@ -10,7 +10,13 @@
 
 <body>
   <noscript>You need to enable JavaScript to view «Logistman» site.</noscript>
-  <div id="root"></div>
+  <div id="root">
+  </div>
+
+  <script
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAdPX1hEy04ED1D8TLhLsTEwNSJ5xbo0Vo&libraries=places">
+  </script>
```

Comment:
I think it's better to move this code to a separate file/component. Let's investigate it...